test/e2e/storage/testsuites/subpath.go

```diff
@@ -684,10 +691,8 @@ func testSubpathReconstruction(f *framework.Framework, pod *v1.Pod, forceDelete
 	utils.TestVolumeUnmountsFromDeletedPodWithForceOption(f.ClientSet, f, pod, forceDelete, true)
 }
 
-func formatVolume(f *framework.Framework, volumeSource *v1.VolumeSource) {
-	var err error
-	// Launch pod to format the volume
-	pod := &v1.Pod{
+func testPodFormatVolume(f *framework.Framework, volumeSource *v1.VolumeSource) *v1.Pod {
```

Comment:
this name is a little confusing (makes it seem like its running a test), maybe: `volumeFormatPod`?

Reply:
done